testsuite/features/build_validation/smoke_tests/smoke_tests.template

```diff
@@ -162,7 +162,7 @@ Feature: Smoke tests for <client>
     When I force picking pending events on "<client>" if necessary
     And I wait until event "Package Install/Upgrade scheduled by admin" is completed
 
-@skip_for_traditional
+@suse_minion
```

Comment:
I approved but then remembered - why `suse_minion` instead of the already existing `sle_minion` tag? is there a difference?